binderhub/app.py

```diff
@@ -165,6 +165,7 @@ def _add_slash(self, proposal):
     repo_providers = Dict(
         {
             'gh': GitHubRepoProvider,
+            'gist': GistRepoProvider,
```

Comment:
Doesn't this add the endpoint?

Reply:
only for the API. If you look at main.py's ParameterizedMainHandler and templates/index.html, you'll see that the code simply dumps it into the appropriate textboxes and triggers 'launch' by fake-clicking the launch button. And if you look at index.js it's hardcoded to use gh....

Reply:
Ah! I hadn't read the code closely enough and assumed that the `ParametrizedMainHandler` kicks off the build :)

Reply:
It's not that smart yet :)

Since this work needs to touch the js, should we finish up and deploy #252 first?